### DISCUSSION TITLE
Deployment of multi-server environment on AWS Elastic Beanstalk using Docker/ECS

### DISCUSSION THREAD

**Discussion by dsturgilljr** [jhuckaby/Cronicle#617] [Q&A]
I am posting looking for some guidance from anyone who has successfully setup a multi-server Cronicle environment using AWS Elastic Beanstalk with Docker/ECS.

I have created two EB environments (workers and primary) with one Cronicle container running in each enviroment. I am up and running with the primary and worker independently, but as soon as I add the hostname name to the server cluster from the UI, the web UI goes into an "endless" refresh loop.

I can provide more information if needed, but I was curious if anyone has dealt with and/or solved this.

**Comment by LohitasyaTata** [comment]
You dont need as master-worker architecture for ECS u can place multiple tasks which can handle load automatically.
https://github.com/LohitasyaTata/cronicleservice.git
